testData/inspections/suppression/fix/innerVariableDeclarationSuppressionFix2-after-highlighting.go

```diff
@@ -1,6 +1,6 @@
 package newmath
 
 func TestFunction() {
-	//noinspection GoUnusedVariable
```

Comment:
There is a way, but this is not the case, I don't see any reason to change inspection name